src/index.html

```diff
@@ -2,16 +2,190 @@
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="8" 
+            maxlength="23"
+            required
+            pattern="/w+"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-groups">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="lovePet" 
+              id="lovePet" 
+              value="yes"
+            >
+            Yes
+          </label>
+          
+          <label>
+            <input 
+              type="radio" 
+              name="lovePet" 
+              id="noLovePet" 
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <label>
+          What is your favorite color?
+          <input 
+            type="color" 
+            name="favoriteColor" 
+            id="favoriteColor"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input 
+            type="time" 
+            name="timeToSleep" 
+            id="timeToSleep"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select 
+            name="carBrand" 
+            id="carBrand" 
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          <div class="Rate">
+            How do you rate our work?
+            <input 
+              type="range" 
+              name="rateOfWork" 
+              id="rateOfWork" 
+              value="0"
+            >
+          </div>
+        </label>
+      </fieldset>
+
+      <fieldset class="form-groups">
+        <legend>Additional info:</legend>
+
+        <label>
+          Comments:
+          <textarea 
+            name="coments" 
+            id="coments" 
+            cols="21" 
+            rows="2" 
+            autocomplete="off"
+         ></textarea>
+        </label>
+
+        <label class="form-field">
+          Would you recomend us?
+          <select name="recomendation" id="recomendation">
+            <option value="yes">Yes</option>
+            <option value="no">No</option>
+          </select>
+        </label>
+      </fieldset>
+      <button class="form-groups" type="submit">Submit</button>
+    </form>
     <script type="text/javascript" src="./main.js"></script>
+    </form>
   </body>
-</html>
+  
+</html>
```

Comment:
[In the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
```suggestion
</html>

```